internal/query/query.go

```diff
@@ -0,0 +1,11 @@
+package query
+
+// FlagParser wraps pflag.FlagSet struct.
+type FlagParser interface {
+	GetBool(string) (bool, error)
+	GetString(string) (string, error)
+	GetStringArray(string) ([]string, error)
+	GetStringToString(string) (map[string]string, error)
+	GetUint(name string) (uint, error)
+	Set(name, value string) error
+}
```

Comment:
Golang is very opinionated on format.  You might want to run gofmt on your files.  I believe it would catch files without newlines like this one and fix them for you.

Reply:
Yeah, not sure how that got left out. It should be running on every save. Eventually I'm going to setup a tool called pre-commit to help prevent this.